include/swift/Option/Options.td

```diff
@@ -508,6 +508,9 @@ def no_static_stdlib: Flag<["-"], "no-static-stdlib">,
 def toolchain_stdlib_rpath: Flag<["-"], "toolchain-stdlib-rpath">,
   Flags<[HelpHidden,DoesNotAffectIncrementalBuild]>,
   HelpText<"Add an rpath entry for the toolchain's standard library, rather than the OS's">;
+def no_toolchain_stdlib_rpath: Flag<["-"], "no-toolchain-stdlib-rpath">,
+  Flags<[HelpHidden,DoesNotAffectIncrementalBuild]>,
+  HelpText<"Do not add an rpath entry for the toolchain's standard library, rather than the OS's">;
```

Comment:
This help text feels weird because of that "rather" clause (copied from the original option). Does it mean the OS's rpath will be added? (No, only if necessary.)

I'm having trouble coming up with a better phrasing, but there probably is one.

Reply:
"Do not add an rpath entry for the toolchain's standard library" alone is probably enough, perhaps with the word "(default)" as well.

Reply:
Done in https://github.com/apple/swift/pull/27207/commits/adce3fb793e80cbd6fb87252f3f7287f95d86d90:

```
"Do not add an rpath entry for the toolchain's standard library (default)"
```